configure.ac

```diff
@@ -64,8 +64,8 @@ AS_IF([test -n "$missing_gmock"],
 AC_SEARCH_LIBS([__b64_ntop], [resolv])
 AC_SEARCH_LIBS([pthread_create], [pthread])
 
-dnl We're pretty crypto-centric, having the OpenSSL libraries in LIBS
-dnl is fine.
+# We're pretty crypto-centric, having the OpenSSL libraries in LIBS
+# is fine.
```

Comment:
Could you slice out the changes to this file to another branch? Keeping the PR to just one thing, and as small as possible...